docs/source/api/notebooks/Notebook_5_GP_solution.ipynb

```diff
@@ -0,0 +1,631 @@
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "## 1. Raw Graph Data Conversion\n",
+    "\n",
+    "Converting the raw graph prediction dataset into `super-node` format for GraphStorm could be illustrated as the diagram below.\n",
+    "\n",
+    "<img src=\"../../../../tutorial/GP_data_conversion.png\" style=\"margin:auto\" width=\"750\" class=\"center\"/>\n",
+    "\n",
+    "In this notebook, we use the [OGBG Molhiv Data](https://ogb.stanford.edu/docs/graphprop/#ogbg-mol), which is a popular graph property prediction dataset. \n",
+    "Due to the large number of lines of code, we will not show the actual raw graph data conversion code in this notebook. Users can find the [souce code of OGBG data conversion](https://github.com/awslabs/graphstorm/blob/main/examples/graph_prediction/gen_ogbg_supernode.py) in GraphStorm's [graph prediction example folder](https://github.com/awslabs/graphstorm/blob/main/examples/graph_prediction/).\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Tip:</b> We also provide a Python script to generate synthetic supernode-based graph data for user to better understand the `super-node` grahp data format, which is avaible <a href=\"https://github.com/awslabs/graphstorm/blob/main/docs/source/api/notebooks/dummy_gc_data.py\"><u>here</u></a>.</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can download the source code of the OGBG conversion and generate the `super-node` format OGBG data with the following commands.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!wget -c https://raw.githubusercontent.com/awslabs/graphstorm/main/examples/graph_prediction/gen_ogbg_supernode.py\n",
+    "\n",
+    "!python gen_ogbg_supernode.py --ogbg-data-name molhiv --output-path ./supernode_raw/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The converted OGBG data will be stored at `./supernode_raw/`. Then we can run GraphStorm's gconstruct command to partition the graph for model training and inference. The processed graph is stored in the `/supernode_gs_1p/` folder."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!python -m graphstorm.gconstruct.construct_graph \\\n",
+    "        --conf-file ./supernode_raw/config.json \\\n",
+    "        --output-dir ./supernode_gs_1p/ \\\n",
+    "        --num-parts 1 \\\n",
+    "        --graph-name supernode_molhiv"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "<b>Tip:</b> If the above Python commands fail to run in a Notebook, please run it in a Linux terminal environment where GraphStorm is install.</div>"
```

Comment:
Do we really need this warning? We can say in the pre-requisite we expect the same Python env to have jupyter and graphstorm installed, e.g. include a command like:

```
# For a Linux system
pip install torch==2.3.0 --index-url https://download.pytorch.org/whl/cpu
pip install dgl==2.3.0  -f https://data.dgl.ai/wheels/torch-2.3/repo.html
pip install graphstorm jupyter`
jupyter notebook ./
```

Reply:
Removed this warning.